dev/service-registry/client-core/src/main/scala/com/lightbend/lagom/internal/registry/ServiceRegistryClient.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal.registry
+
+import java.net.URI
+
+import scala.concurrent.Future
+
+private[lagom] trait ServiceRegistryClient {
+  def locateAll(serviceName: String): Future[List[URI]]
```

Comment:
List is lagom style? Just checking, we usually do immutable.Seq, either works, no worries

Reply:
Let's converge, then.